x-pack/plugins/ml/public/alerting/register_ml_alerts.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { lazy } from 'react';
+import { MlStartDependencies } from '../plugin';
+import { ML_ALERT_TYPES } from '../../common/constants/alerts';
+
+export function registerMlAlerts(
+  alertTypeRegistry: MlStartDependencies['triggersActionsUi']['alertTypeRegistry']
+) {
+  alertTypeRegistry.register({
+    id: ML_ALERT_TYPES.ANOMALY_DETECTION,
+    description: i18n.translate('xpack.ml.alertTypes.anomalyThreshold.description', {
+      defaultMessage: 'Alert when anomaly detection jobs results match the condition.',
+    }),
+    iconClass: 'bell',
+    documentationUrl(docLinks) {
+      // TODO add the documentation link
+      return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/kibana/${docLinks.DOC_LINK_VERSION}/ml-alerts.html`;
+    },
+    alertParamsExpression: lazy(() => import('./ml_anomaly_alert_trigger')),
+    validate: () => ({
+      errors: [],
+    }),
+    requiresAppContext: false,
+    defaultActionMessage: i18n.translate(
+      'xpack.ml.alertTypes.anomalyThreshold.defaultActionMessage',
+      {
+        defaultMessage: `Elastic Stack Machine Learning Alert:
+- Job IDs: \\{\\{#context.jobIds\\}\\}\\{\\{context.jobIds\\}\\} - \\{\\{/context.jobIds\\}\\}
```

Comment:
Got this error after creating an alert:

![image](https://user-images.githubusercontent.com/7405507/107036829-20f99800-67b2-11eb-9388-b97b1d7f12f2.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/89286/commits/de56b53a082dc62d5fcda1c4ffbf2d6f855559a3